resources/views/layouts/base.blade.php

```diff
@@ -32,12 +32,12 @@ class="scroll-smooth"
     $user = auth()->user();
 @endphp
 
-<nav class="bg-main z-10 p-4 bg-gradient-to-r from-main to-main-light">
+<nav dusk="main-header" class="bg-main z-10 p-4 bg-gradient-to-r from-main to-main-light">
     <div
         class="container flex justify-between text-white gap-4 items-center m-auto relative px-2"
         x-data="{ open: false }"
     >
-        <div class="dark:text-gray-200 text-lg md:text-xl font-bold relative z-20">
+        <div dusk="header-logo" class="dark:text-gray-200 text-lg md:text-xl font-bold relative z-20">
```

Comment:
Shouldn't we use `@dusk('header-logo)`. I kind of thought Laravel had this built-in, but apparently it hasn't? 🤔

The idea would be that `@dusk` will only write the `x-dusk` attribute when not in production, so that we don't have additional dusk attributes on the real site.

Reply:
Hmm, I'll try to look more into this, I haven't found anything here https://laravel.com/docs/10.x/dusk#dusk-selectors